tests/test_classifier/test_class_prediction_error.py

```diff
@@ -108,3 +110,26 @@ def test_class_type(self):
         with self.assertRaises(YellowbrickValueError):
             visualizer = ClassPredictionError(model)
             visualizer.score(X, y)
+
+    def test_score_returns_score(self):
+        """
+        Test that ClassPredictionError score() returns a score between 0 and 1
+        """
+        data = self.load_data("occupancy")
+        X = data[[
+            "temperature", "relative_humidity", "light", "C02", "humidity"
+        ]]
+
+        y = data['occupancy']
+
+        # Convert X to an ndarray
+        X = X.copy().view((float, len(X.dtype.names)))
+
+        X_train, X_test, y_train, y_test = tts(X, y, test_size=0.2, random_state=42)
```

Comment:
For this test, it might be simpler to use the [`X` and `y` variables created on L38](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_classifier/test_class_prediction_error.py#L38) if you wanted - but this is perfectly fine. `ClassPredictionErrorTests` does need to have an integration test with a real data set + pandas added (though that's not for the PR, that's just a general comment).

Reply:
Done